versions.rst

```diff
@@ -41,6 +41,9 @@ Unsupported versions
 Status key
 ==========
 
+.. note:: Documentation and test changes may be backported to any branch in a release
+    cycle above **bugfix**.
```

Comment:
Can you add a link somehow to "Maintenance branches" section?

Reply:
Jacob -- I'd delete this note, and add a few words to the 'see also' below on backporting.

Reply:
This was marked as resolved but the text remains?